doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+    fi
+
+    echo "Sos process done!"
+    echo " ***** Please reboot device to test on new sos. *****"
+else
+    echo -e "Fail to upgrade sos $target_clearlinux from mirror url."
+fi
+
+echo "Init swupd mirror & proxy"
+swupd mirror -u
+unset https_proxy
+umount /mnt
+}
+
+function upgrade_uos()
+{
+# Add proxy if you need to download uos img automatically.
+if [[ -n $proxy ]]; then
+    echo "Add proxy..."
+    export https_proxy=$proxy
+fi
+
+# Do upgrade uos process.
+if [[ $skip_download_uos == 1 ]]; then
+    uos_img_xz_path=$(find ~/ -name clear-$target_version-kvm.img.xz)
+    uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+    if [[ ! -f $uos_img_xz_path ]] && [[ ! -f $uos_img_path ]]; then
+        echo "You should download uos clear-$target_version-kvm.img.xz file firstly." && exit
+    fi
+    if [[ -f $uos_img_xz_path ]]; then
+        echo "get uos xz file in: $uos_img_xz_path"
+        unxz $uos_img_xz_path
+        uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+    fi
+    echo "get uos img file in: $uos_img_path"
+    if [[ $? -eq 0 ]]; then
+        losetup -f -P --show $uos_img_path
+    else
+        echo "fail to losetup uos img, please check uos img status" && exit 1
+    fi
+    mount /dev/loop0p3 /mnt
+    cp -r /usr/lib/modules/"`readlink /usr/lib/kernel/default-iot-lts2018 | awk -F '2018.' '{print $2}'`.iot-lts2018" /mnt/lib/modules
+    umount /mnt
+    sync
+else
+    cd ~
+    curl https://download.clearlinux.org/releases/$target_version/clear/clear-$target_version-kvm.img.xz -o clear-$target_version-kvm.img.xz
+    if [[ $? -eq 0 ]]; then
+        unxz clear-$target_version-kvm.img.xz
+    fi
+    if [[ $? -eq 0 ]]; then
+        losetup -f -P --show clear-$target_version-kvm.img
+    fi
+    mount /dev/loop0p3 /mnt
+    cp -r /usr/lib/modules/"`readlink /usr/lib/kernel/default-iot-lts2018 | awk -F '2018.' '{print $2}'`.iot-lts2018" /mnt/lib/modules
+    umount /mnt
+    sync
+fi
+uos_img_path=$(find ~/ -name clear-$target_version-kvm.img)
+cp -r /usr/share/acrn/samples/nuc/launch_uos.sh ~/
+sed -i "s/\(virtio-blk.*\)\/home\/clear\/uos\/uos.img/\1$(echo $uos_img_path | sed "s/\//\\\\\//g")/" ~/launch_uos.sh
+echo "Upgrade uos done..."
+echo "Now you can run below command to start UOS..."
+echo "# cd ~/ && ./launch_uos.sh -V 1"
+}
+
+# Set script options.
+while getopts "t:p:m:huso" opt
+do
+        case "$opt" in
+                t) target_version="$OPTARG"
+                        ;;
+                p) proxy="$OPTARG"
+                        ;;
+                m) mirror="$OPTARG"
+                        ;;
+                u) upgrade_uos=1
+                        ;;
+                s) skip_download_uos=1
+                        ;;
+                h) print_help
+                        ;;
+                ?) print_help
+                        ;;
+        esac
+done
+
+# -t opt is must
+if [[ -z $1 ]]; then
```

Comment:
We should add a check to ensure we have root privileges:
```suggestion
if [[ -z $1 || $EUID -ne 0 ]]; then
```